avm/src/vm/memory.rs

```diff
@@ -793,22 +794,25 @@ impl HandlerMemory {
       }
     }
     // Finally pull any addresses added by the old object into the new with a similar stitching
-    /*if hm.addr.0.len() > self.addr.0.len() {
```

Comment:
Nice, I think @dfellis had commented this code out for a particular reason but I'm not sure what it was